lms/templates/verify_student/incourse_reverify.underscore

```diff
@@ -0,0 +1,31 @@
+<div id="wrapper-facephoto" class="wrapper-view block-photo face-photo-step">
+  <div class="facephoto view">
+    <h3 class="title"><%- gettext( "Take Your Photo" ) %></h3>
+    <div class="instruction">
+      <p><%= gettext("Course key: ") + courseKey %></p>
+      <p><%= gettext("Checkpoint name: ") + checkpointName %></p>
```

Comment:
@srpearce What instructional text should we include for submitting a face photo for re-verification?